src/app/llms.txt/route.ts

```
@@ -1,49 +1,43 @@
+    bySection.set(section, list);
+  }
+
+  const lines: string[] = ['# Novu Documentation', '', '> Notification infrastructure for developers.', ''];
+
+  for (const [section, items] of bySection) {
+    lines.push(`## ${SECTION_LABELS[section] ?? section}`, '');
+    items.sort((a, b) => a.url.localeCompare(b.url));
+    for (const page of items) {
+      const title = page.data.pageTitle ?? page.data.title;
+      const rawDesc =
+        typeof page.data.description === 'string' ? page.data.description : '';
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)